src/components/app/app.scss

```diff
@@ -1,11 +1,15 @@
+body {
+  font-family: "Helvetica", sans-serif;
+}
+
 /**
  * Sample Style
  */
 $steps: 30;
 @keyframes rainbow-shift {
   @for $i from 0 through $steps {
     #{percentage($i * (1 / $steps))} {
-      color: hsla(random(360), 100, 50, 1)
+      color: hsla(random(360), 100, 50, 1);
```

Comment:
should we add prettier checks for CSS to the kanban board?

Reply:
My prettier actually formatted this, so I'm not sure if that's already configured or not 🤔